src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -51,27 +51,27 @@ namespace OpenMS
   {
     std::vector<std::string> true_false = {"true","false"};
 
-    defaults_.setValue("method", "top", "- top - quantify based on three most abundant peptides (number can be changed in 'top').\n- iBAQ (intensity based absolute quantification), will override top with 0 and average with sum.");
+    defaults_.setValue("method", "top", "- top - quantify based on three most abundant peptides (number can be changed in 'top').\n- iBAQ (intensity based absolute quantification), calculate the sum of all peptide peak intensities divided by the number of theoretically observable tryptic peptides (https://rdcu.be/cND1J).");
     defaults_.setValidStrings("method", {"top","iBAQ"});
 
-    defaults_.setValue("top", 3, "Calculate protein abundance from this number of proteotypic peptides (most abundant first; '0' for all)");
-    defaults_.setMinInt("top", 0);
+    defaults_.setValue("top:N", 3, "Calculate protein abundance from this number of proteotypic peptides (most abundant first; '0' for all)");
+    defaults_.setMinInt("top:N", 0);
 
-    defaults_.setValue("average", "median", "Averaging method used to compute protein abundances from peptide abundances");
-    defaults_.setValidStrings("average", {"median","mean","weighted_mean","sum"});
+    defaults_.setValue("top:average", "median", "Averaging method used to compute protein abundances from peptide abundances");
```

Comment:
I think if we start renaming, we should call this "top:aggregate" or something like that. A sum is not an average